test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -335,12 +336,16 @@ export default class PerfCompare extends Component {
 
             const [baselineSchemeWithHostWithPort, baselineBuildNum] = this.state.inputURL.baselineID.split("build_id=");
             const [testSchemeWithHostWithPort, testBuildNum] = this.state.inputURL.testID.split("build_id=");
-
+                                      
             // Check if the benchmark and test data is valid
-            if (baselineBuildNum === undefined || (baselineSchemeWithHostWithPort.substring(0,31) !== "http://perffarmServer")) {
+            if ((baselineSchemeWithHostWithPort.substring(0, server.length) !== server)){
+           		 displayErrorMessage += "Invalid ServerID URL. Please refer to the trscConfig.json file in PerfCompare folder. "
```

Comment:
This error comment needs to be updated since we aren't using the `trscConfig.json` any more.